clusterloader2/pkg/measurement/common/latencymetric.go

```diff
@@ -0,0 +1,39 @@
+/*
```

Comment:
Since this is not a measurement itself (but more general util for measurements) and IIUC you're placing measurement implementations under `common/`, probably it makes sense to move this to somewhere like `measurement/util/`?

Reply:
`measurement/util` directory created. I moved there all files related to measurements that doesn't contain metric implementation by itself.